integration/messaging/test/static/firebaseConfig.js

```diff
@@ -15,19 +15,20 @@
  * limitations under the License.
  */
 
-const config = {
-  apiKey: 'AIzaSyCaSnFca0oG_b-t-s_p9wRdr9o9aSPmolE',
-  authDomain: 'fcm-sdk-testing-vapid-key.firebaseapp.com',
-  databaseURL: 'https://fcm-sdk-testing-vapid-key.firebaseio.com',
-  projectId: 'fcm-sdk-testing-vapid-key',
-  storageBucket: '',
-  messagingSenderId: '650229866790'
+const FIREBASE_CONFIG = {
+  apiKey: 'AIzaSyBVqFz5vuknTmaVJyRnjxSolqhc8hXx0M4',
```

Comment:
Who owns this project?

Reply:
this is now using the test project that the FCM team shares. Looks like it's currently shared with 70+ firebase folks including devRel, test and backend folks. I added you and Christina as editor just now in case you needed to access 🍻